Sources/Charcoal/Filters/List/ListFilterViewController.swift

```diff
@@ -33,6 +33,7 @@ final class ListFilterViewController: FilterViewController {
     init(filter: Filter, selectionStore: FilterSelectionStore) {
         self.filter = filter
         super.init(title: filter.title, selectionStore: selectionStore)
+        scrollView = tableView
```

Comment:
Is it possible to avoid setting this in `init` because it breaks lazy initialization of `tableView`?

Reply:
I'll see if I can move it somewhere else